diff --git a/scripts/resources.js b/scripts/resources.js
index 0611d8f9a39552c79d588b27c7460df609d722b2..aaf9c3fba147edeff9109f1c658a41197deacba3 100644
--- a/scripts/resources.js
+++ b/scripts/resources.js
@@ -80,7 +80,7 @@ async function downloadAndUnpackResource(uri, destPath, callback) {
           });
     } else if (uri.endsWith('.tar.gz')) {
       response.on('data', chunk => bar.tick(chunk.length))
-          .pipe(tar.x({C: destPath, strict: true}))
+          .pipe(tar.x({C: destPath, strict: false}))
           .on('close', () => {
             if (callback !== undefined) {
               callback();
